src/Language/GraphQL/Draft/Syntax.hs

```diff
@@ -374,56 +380,57 @@ isNotNull = not . isNullable
 
 -- * Type definition
 
-data TypeDefinition possibleTypes
+data TypeDefinition possibleTypes a
```

Comment:
How would you feel about replacing `a` with a more meaningful name that indicates what role it plays, just like we have `possibleTypes`? (Genuine question.)

Reply:
Your suggestion makes sense. The `a` here is representing the type of input value. So, I think `inputValue` would be a meaningful name here.

Reply:
I leave this up to you to decide as you wish. I'm not against merging this PR as-is (which is why I approved it).